extensions/cms/src/cmsUtils.ts

```diff
@@ -167,38 +241,83 @@ export class CmsUtils {
 
 	public async removeRegisteredServer(registeredServerName: string, relativePath: string, ownerUri: string): Promise<boolean> {
 		let provider = await this.getCmsService();
-		return provider.removeRegisteredServer(ownerUri, relativePath, registeredServerName).then((result) => {
-			return result;
-		});
+		let result = await provider.removeRegisteredServer(ownerUri, relativePath, registeredServerName);
+		return result;
 	}
 
 	public async addServerGroup(groupName: string, groupDescription: string, relativePath: string, ownerUri: string): Promise<boolean> {
 		let provider = await this.getCmsService();
-		return provider.addServerGroup(ownerUri, relativePath, groupName, groupDescription).then((result) => {
-			return result;
-		});
+		let result = await provider.addServerGroup(ownerUri, relativePath, groupName, groupDescription);
+		return result;
 	}
 
 	public async removeServerGroup(groupName: string, relativePath: string, ownerUri: string): Promise<boolean> {
 		let provider = await this.getCmsService();
-		return provider.removeServerGroup(ownerUri, relativePath, groupName).then((result) => {
-			return result;
-		});
+		let result = await provider.removeServerGroup(ownerUri, relativePath, groupName);
+		return result;
+	}
+
+	public static getConnectionProfile(connection: azdata.connection.Connection): azdata.IConnectionProfile {
+		let connectionProfile: azdata.IConnectionProfile = {
```

Comment:
Having all these static methods scattered about is a bit odd - would suggest putting them all in one place so it's easy to separate from the instance functions.